src/modules/SearchPage/components/TypeCard/styled.ts

```diff
@@ -1,10 +1,12 @@
 import styled from 'styled-components';
 
-export const Card = styled.div`
+export const Card = styled.button`
```

Comment:
same

Reply:
Is styles for this cards are du/three-plicated?

Reply:
Yes, changed